docker-compose.yml

```diff
@@ -3,19 +3,19 @@ services:
   db:
     container_name: respa-db
     build: ./docker/postgres/
-    restart: always
+    restart: on-failure
     environment:
```

Comment:
I don't understand the ramifications for this.  Is it a developer help to not ditch the db contents upon docker-compose stop & docker-compose up?

Reply:
this is not about the contents, this is about the container that does not shut down ever, unless you manually locally change the restart policy, which is super annoying

Reply:
also this restart policy starts the container automatically upon the launch of docker on the machine, which leads to the port being occupied if you then decide to start another project, where db container uses the same port